### DISCUSSION TITLE
Move add-on config to the add-on config

### DISCUSSION THREAD

**Discussion by frenck** [home-assistant/frontend#5708] [Supervisor]
Recently a new tabbed interface was introduced in the supervisor. On add-ons that gives a separate tab for the add-on configuration.

However, sneaky, some settings are still on the "Info" tab.

![image](https://user-images.githubusercontent.com/195327/80860622-39f47c00-8c69-11ea-91d4-a8854de1c7dc.png)

I think these should be moved into the "Configuration" tab as well.

**Comment by KTibow** [comment]
I think renaming the "Info" tab would be better. Users might want to change this setting as soon as they install the add-on.
